software/teachee-desktop/src/usb_manager.rs

```diff
@@ -0,0 +1,14 @@
+use std::{
+    sync::{Arc, Mutex},
+    thread,
+    time::Duration,
+};
+
+use crate::storage::Storage;
+
+pub fn usb_manager(storage: Arc<Mutex<Storage>>) {
```

Comment:
`usb_manager` should have fields which are not behind a lock, just like App. See previous comment.

Reply:
Updated with struct, the struct is created in App::new and moved to usb_manager loop